eng/testing/linker/SupportFiles/Directory.Build.targets

```diff
@@ -1,18 +1,20 @@
 <Project>
   <Target Name="RestoreProject">
     <MSBuild Projects="$(MSBuildProjectFullPath)"
-             Properties="Configuration=$(Configuration)"
+             Properties="Configuration=$(Configuration);ForceEvaluation=true"
              Targets="Restore" />
   </Target>
 
-  <Target Name="AddFrameworkReference">
+  <Target Name="RemoveRuntimePackFromDownloadItem"
+          AfterTargets="ProcessFrameworkReferences">
```

Comment:
NIT: is there a reason why we chose AfterTargets instead of DependsOn? That way bellow you could make PublishTrimmed to depend on this one as opposed to ProcessFrameworkReferences. The advantage of DependsOn over AfterTargets is that AfterTargets will always run (even multiple times) whereas DependsOn would only run once.

Reply:
An MSBuild target is never invoked more than once. AfterTargets doesn't change that and should be preferred usually over DependsOn which is an old relict before AfterTargets was introduced.

Reply:
What @ViktorHofer said 😄